### DISCUSSION TITLE
Bump jose to v5

### DISCUSSION THREAD

**Discussion by novotnyJiri** [panva/openid-client#694] [Ideas]
According to https://www.npmjs.com/package/jose `jose` v4 no longer supports bug fixes and new features. Have you thought about upgrading? Our team uses `node-openid-client` and we would like to upgrade our codebase to v5.

**Comment by panva** [comment]
The supported engines of openid-client@5 and jose@5 don't align which is why it wasn't upgraded. `openid-client` is due for a complete rewrite soon anyway, new API, less dependencies, more runtime compatibility, tree-shakeable ESM, fully typed, no singleton side-effects.

**Reply by novotnyJiri** [reply]
Great! Thanks for answer. Any ETA when the new version is released? And GL with coding

**Reply by panva** [reply]
Not really, no.

**Reply by novotnyJiri** [reply]
Alright, thanks for quick response.

**Reply by panva** [reply]
See https://github.com/panva/node-openid-client/tree/next

I'm at a point where I need to write documentation and examples and that's not fun so I'm lacking motivation.

**Reply by panva** [reply]
https://github.com/panva/openid-client/releases/tag/v6.0.0

**Reply by novotnyJiri** [reply]
Great!
